### ISSUE TITLE
 Support for `cp`, `rm`, and `mv` like sub commands

### ISSUE THREAD

**Issue by isaacazuelos** [isaacazuelos/keep#7]
Odds are these will all be pretty easy to implement once the infrastructure is in place.

They should all be pretty low priority as well, since the store is just a directory.

`keep cp old new` would just be `cp (keep path old) (keep path new)` in some sense.

**Comment by isaacazuelos** [comment]
Closed by 7c8c92fd69ae6187350d6455622b556c5d7af828.

We still need a good way to cover the case where `old` and `new` are directories, for `cp` and `mv`.

**Comment by isaacazuelos** [comment]
These are temporarily disabled as they're not very useful, and there are incoming changes to the way commands are implemented.
